From e1a6dbd0647c8747ad213e9ea561b034b58fe3ef Mon Sep 17 00:00:00 2001
From: "ayyappu.varunkumar" <ayyappu.varunkumar@lgepartner.com>
Date: Thu, 25 Mar 2021 17:50:08 +0530
Subject: [PATCH] Add oss-pkg-info yaml

:Release Notes:
Add oss yaml file information

:Detailed Notes:
To remove build warning & proper
license information added yaml

:Testing Performed:
Build verified.

:QA Notes:
N/A

:Issues Addressed:
[PLAT-133040] add oss-pkg-info yaml for ls2-helpers

Change-Id: I5031440e9bf88a8368f664915ed2ac821f696054
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/ls2-helpers/+/348046 Add oss-pkg-info yaml]

 oss-pkg-info.yaml | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 oss-pkg-info.yaml

diff --git a/oss-pkg-info.yaml b/oss-pkg-info.yaml
new file mode 100644
index 0000000..a25a1b0
--- /dev/null
+++ b/oss-pkg-info.yaml
@@ -0,0 +1,4 @@
+Open Source Software Package:
+- name: webOSOSE-ls2-helpers
+  source: https://github.com/webosose/ls2-helpers
+  license: Apache-2.0
